Headers/DebugServer2/Target/ThreadBase.h

```diff
@@ -57,6 +57,9 @@ class ThreadBase {
                            Address const &address = Address()) = 0;
 
 public:
+  virtual ErrorCode beforeResume();
```

Comment:
You might be right. Do we disable them when a thread stops?

On Tuesday, August 9, 2016, Francis Ricci <notifications@github.com> wrote:

> In Headers/DebugServer2/Target/ThreadBase.h
> <https://github.com/facebook/ds2/pull/243#discussion_r74099590>:
>
> > @@ -57,6 +57,9 @@ class ThreadBase {
> >                             Address const &address = Address()) = 0;
> >
> >  public:
> > +  virtual ErrorCode beforeResume();
>
> I don't think we'd even have to check if it's kReasonThreadEntry.
> Shouldn't breakpoints always be re-enabled when resuming?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/243/files/2a73b10db4691d10fd51d07ec46fbded7b36f6af..b32f1128fa8ceb0010cf5f79262d062aca477162#r74099590>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbB3Iyn6Uwv2cQz9iOuQM5G8Jq9VOks5qeLJDgaJpZM4H9FRy>
> .
>


-- 
Stephane Sezer